### PR TITLE
Add Devfolio to swag list

### PR DESCRIPTION
**Pull request by code-monk08**
Happy Hacktoberfest!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,6 +312,16 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Notes**:  See this page for their [FAQ's](https://substrate.dev/hacktoberfest/faq/) or this page for their [official announcement](https://www.parity.io/hacktoberfest-2019-parity/).
 
 
+### 4 Merged PRs
```
Comment (crweiner) [inline README.md]: No real need to make a new level for 4 merged PRs, you can just add it to the **2 or more merged PR** section

File: README.md [hunk 2]
```diff
@@ -303,6 +312,16 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
+
+#### **Devfolio** (limited edition t-shirts and stickers)
+
+- **Requirements**: Get atleast four pull request merged into any project [react-otp-input](https://github.com/devfolioco/react-otp-input), [devfolio-gatsby-starter](https://github.com/devfolioco/devfolio-gatsby-starter) or [react-intl-tel-input](https://github.com/patw0929/react-intl-tel-input) to win stickers and for four exceptional pull requests get a chance to earn their LIMITED EDITION t-shirts!
```
Comment (crweiner) [inline README.md]: Their blog post lists [4 repos](https://devfolio.co/blog/hacktoberfest-2019-devfolio/) and please double check your spelling here. :wink:

File: README.md [hunk 3]
```diff
@@ -60,6 +60,15 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: [Join #Orbtoberfest](https://circleci-public.github.io/hacktoberfest/#/) and submit your PRs to this [Google Form](https://docs.google.com/forms/d/1-MbzLy0m1eRLjeye9wu76fYkzcX5A3PvgKd09EhVObY/viewform).
 - **Notes**: View full details above. Make pull requests to any open-source "orbs" for CircleCI, or create your own. When complete submit PRs to the Google form to claim swag.
 
+### D
+
+#### **Devfolio**
+
+- **Swag**: Stickers, Limited Edition T-Shirts
+- **Requirements**: Get atleast four pull request merged into any project [react-otp-input](https://github.com/devfolioco/react-otp-input), [devfolio-gatsby-starter](https://github.com/devfolioco/devfolio-gatsby-starter) or [react-intl-tel-input](https://github.com/patw0929/react-intl-tel-input) to win stickers and for four exceptional pull requests get a chance to earn their LIMITED EDITION t-shirts!
```
Comment (crweiner) [inline README.md]: Their blog post lists [4 repos](https://devfolio.co/blog/hacktoberfest-2019-devfolio/) and please double check your spelling here. :wink:

**Comment by crweiner** [comment]
Hi @code-monk08! Thanks for adding another great organization to the Hacktoberfest Swag List!! 

I'd appreciate it if you could please review my comments above and fix the merge conflicts so I can take another look over it and hopefully get it merged. :+1:

File: README.md (hunk 1 above)
Comment (code-monk08) [reply README.md]: Sure.

**Comment by crweiner** [comment]
Marvelous! Thank you again for another wonderful contribution and have a great Hacktoberfest!! :+1: :jack_o_lantern: :smile:

**Pull request merged**